src/Ide/Cradle.hs

```diff
@@ -470,6 +470,7 @@ cabalHelperCradle file = do
                                           , componentRoot = cwd
                                           , componentDependencies = []
                                           }
+                                , runGhcCmd = \_ -> pure CradleNone
```

Comment:
`runGhcCmd` should mirror the behaviour of hie-bios's built in direct cradle and call `ghc`